packages/react/src/components/koros/Koros.tsx

```diff
@@ -36,6 +36,16 @@ export type KorosProps = {
    * Additional style properties to apply to component
    */
   style?: React.CSSProperties;
+  /**
+   * Shift koros up/down for the height of the wave/shape.
+   * Useful when koros should cover part of the adjacent element
+   * Koros is shifted to the direction of the koros shape
+   */
+  shift?: boolean;
```

Comment:
I 'm not sure understand what this does 😆 Does this cause the Koros to overlap with an element above/under it? If so, then it needs `flipHorizontal` to be set as well sometimes? And if so, then maybe that could be mentioned.

Reply:
Without this the gaps in the Koros shape shows the background color, not the image below/above it. This property pulls (with negative-margin) the adjacent element/image and the element/image is rendered below the Koros.

Could also be done by adding extra elements and setting `position: absolute` to the container of the koros. But anyway the koros height must be calculated and it varies with the shape.

I added the positioning and height adjustment to the Koros component itself, so those are re-usable and not Hero specific.

The new "compact" property cuts the Koros, so it is not  85px high, just exactly the height of the shape/pattern of the Koros.

Reply:
Added better descriptions to stories.

Reply:
I was also thinking would it make it obsolete if the heights of the koros could be exported somehow 🤔

Reply:
Changed